src/lib389/setup.py

```diff
@@ -59,19 +72,19 @@
 
     # find lib389/clitools -name ds\* -exec echo \''{}'\', \;
     data_files=[
-        ('/usr/sbin/', [
+        (prefix+'/sbin/', [
```

Comment:
I think `os.path.join()` should be used for paths here

Reply:
Technically right, but there is not much interest as we do not support (nor plan to support) Windows anyway. Furthermore prefix+'/sbin/' is used in the main branch ...